### PR TITLE
Create houzz.go

### PR DESCRIPTION
**Pull request by Real4XoR**

### CODE REVIEW

File: importers/houzz.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+)
+
+type TemplateLineParser struct{}
+
+func (t TemplateLineParser) ParseLine(line string) ([]interface{}, error) {
+	entries := make([]interface{}, 0)
+	// split a line in the text file
+	data := util.SplitString(line, ' ', true, true)
+	// check data here
+	if len(data) > 2 || strings.Contains(data[3], "SEC"{
```
Comment (SoftPoison) [inline importers/houzz.go]: I think you're missing a `)` here

**Review by SoftPoison — CHANGES_REQUESTED** [review]
Just one thing to fix up. Please also double check that the file is formatted properly with `gofmt importers/houzz.go`. Other than that, LGTM :^)

File: importers/houzz.go (hunk 1 above)
Comment (Real4XoR) [reply importers/houzz.go]: Oh cool I didn't even know that existed. Will fix it now :)

File: importers/houzz.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+)
+
+type TemplateLineParser struct{}
+
+func (t TemplateLineParser) ParseLine(line string) ([]interface{}, error) {
+	entries := make([]interface{}, 0)
+	// split a line in the text file
+	data := util.SplitString(line, '\s', true, true)
+	// check data here
+	if len(data) > 2 || strings.Contains(data[3], "SEC"){
```
Comment (SoftPoison) [inline importers/houzz.go]: I think your logic here is inverted. I think you want:
```
if len(data) < 5 || !strings.Contains(data[3], "SEC") {
```

Did you test that this importer works correctly?

**Review by SoftPoison — CHANGES_REQUESTED** [review]

File: importers/houzz.go (hunk 2 above)
Comment (Real4XoR) [reply importers/houzz.go]: Ah yup that's my bad, added those checks after I tested it and didn't retest. I'll fix it up now

**Comment by bogannz** [comment]
Can we also please do this wee search & replace on the data set
`sed -E 's/\\_\\_SEC\\_\\_..(.{16})/$6$\1$/' <file>`

**Comment by Real4XoR** [comment]
Alright fixed the importer and tested properly to confirm everything works. I've also run the sed command on the text file and re-uploaded to the Steamer DB.

**Review by SoftPoison — APPROVED** [review]
Good to merge!

**Pull request merged**